aircraftlib/aircraft.capnp.go

```diff
@@ -27,11 +26,18 @@ func (s Zdate) MarshalJSON() (bs []byte, err error) { return }
 
 type Zdate_List C.PointerList
 
-func NewZdateList(s *C.Segment, sz int) Zdate_List { return Zdate_List(s.NewUInt32List(sz)) }
+func NewZdateList(s *C.Segment, sz int) Zdate_List { return Zdate_List(s.NewCompositeList(8, 0, sz)) }
```

Comment:
It's outside of go-capnproto; it's a change to `capnp compile`.  Cap'n Proto 0.5.0 marks any struct as having a `preferredListEncoding` of `inlineComposite`.